src/lib/snykGroup.ts

```diff
@@ -25,6 +25,7 @@ export class snykGroup {
   private _snykMembershipQueue: any[];
   private _snykMembershipRemovalQueue: any[];
   private _buffer: number = 250;
+  private _roles: any = {};
```

Comment:
Why dictionary and not a list?

Reply:
this is a key vaue pairs that map role names to role id's which allows me to make the relevant api calls using just a role name. maybe I should rename it to something like _rolemapping?